src/main/java/com/sforce/ws/ConnectorConfig.java

```diff
@@ -549,4 +573,27 @@ public void setCaptureHtmlExceptionResponse(boolean captureHtmlResponse) {
     public MessageCaptureHandler getCaptureHtmlHandler() {
         return this.captureHtmlHandler;
     }
+
+    /**
+     * Check if the provided java version has been patched for the bug JDK-8209178. Refer
+     * <a href="https://bugs.openjdk.org/browse/JDK-8209178">JDK-8209178</a> to get the list of java versions that have
+     * fixed the bug.
+     *
+     * @param runtimeVersion java version
+     * @return true, if the java version has been patched for the bug, false otherwise
+     */
+    public static boolean doesVersionHasABug(String runtimeVersion) {
```

Comment:
The name of this method sounds odd. Maybe just "versionHasABug()) would be better.